src/components/Books.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import './Main.css';
+import './Book.css';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import Header from './Header';
+import 'react-circular-progressbar/dist/styles.css';
+
+function Book({
```

Comment:
[OPTIONAL] I think you should move this to another file. This helps readability and ensures separation of concerns.